internal/transform/utils.go

```diff
@@ -0,0 +1,17 @@
+package transform
+
+import (
+	astro "github.com/snowpackjs/astro/internal"
+)
+
+func hasTruthyAttr(n *astro.Node, key string) bool {
+	for _, attr := range n.Attr {
+		if attr.Key == key &&
+			(attr.Type == astro.EmptyAttribute) ||
+			(attr.Type == astro.ExpressionAttribute && attr.Val == "true") ||
+			(attr.Type == astro.QuotedAttribute && (attr.Val == "" || attr.Val == "true")) {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
This is a shared function for checking things like:
```
<style global />
<style global={true} />
<style global="true" />
<style global="" />
<script hoist />
<script hoist={true} />
<script hoist="true" />
<script hoist="" />
```

Reply:
That’s pretty cool we retain HTML exactly as-written!